.github/pull_request_template.md

```diff
@@ -4,5 +4,6 @@ If some of the following don't apply, please write a short explanation why.
 
 - [ ] Changes file added (if needed)
 - [ ] Documented any API changes
+- [ ] Documented any permissions/policy changes
```

Comment:
```suggestion
- [ ] Documented any permissions changes
```
Does the item still effectively communicate what needs to be done without the word "policy" here? My concern is the potential for confusion with the Policy feature in Fleet.

Reply:
Fair!